template/src/ontology/Makefile.jinja2

```diff
@@ -444,7 +444,7 @@ $(IMPORTDIR)/{{ ont.id }}_import.owl: {% if project.import_group.use_base_mergin
 		remove $(patsubst %, --term %, $(ANNOTATION_PROPERTIES)) -T $(IMPORTDIR)/{{ ont.id }}_terms_combined.txt --select complement --select "classes individuals annotation-properties" \
 		$(ANNOTATE_CONVERT_FILE); fi
 {% elif ('custom' == ont.module_type) or (ont.module_type is none and 'custom' == project.import_group.module_type) %}
-$(IMPORTDIR)/{{ ont.id }}_import.owl: {% if project.import_group.use_base_merging %}$(MIRRORDIR)/merged.owl{% else %}$(MIRRORDIR)/{{ ont.id }}.owl{% endif %}
+$(IMPORTDIR)/{{ ont.id }}_import.owl: {% if project.import_group.use_base_merging %}$(MIRRORDIR)/merged.owl{% elif 'none' != ont.mirror_type %}$(MIRRORDIR)/{{ ont.id }}.owl{% endif %}
```

Comment:
So what should happen here if mirror type is none? Does this mean there  _is no mirror_? If so its risky to make this default!

Reply:
Yes, none means no mirror. Can you detail it more?

Reply:
This is the main reason for the issue we want to fix, no?

Reply:
If there isn't a mirror, it won't add the mirror file.